ui/app/templates/components/dashboard/quick-actions-card.hbs

```diff
@@ -0,0 +1,63 @@
+<h3 class="title is-4">Quick actions</h3>
+
+<div class="has-bottom-margin-m">
+  <h4 class="title is-6">Secrets engines</h4>
+  <SearchSelect
+    @id="secrets-engines-select"
+    @options={{this.mountOptions}}
+    @selectLimit="1"
+    @disallowNewItems={{true}}
+    @fallbackComponent="input-search"
+    @onChange={{this.handleSearchEngineSelect}}
+    @placeholder="Type to select a mount..."
+    @displayInherit={{true}}
+    @shouldRenderName={{true}}
+    @passObject={{true}}
+    @objectKeys={{array "type"}}
+    class="is-marginless"
+  />
+</div>
+
+{{#if this.selectedEngine}}
+  <h4 class="title is-6">Action</h4>
+  <Select
+    @name="auth-method"
```

Comment:
I think `@name` here might be copy pasta'd 🍝  - can be updated to something like `action-select`?